Makefile

```diff
@@ -325,7 +379,9 @@ clean:
 	cd Firefox && $(MAKE) clean
 	cd Webmin && $(MAKE) clean
 	cd Chromium && $(MAKE) clean
-	rm -f scap-security-guide.spec
+	rm -f $(RPM_SPEC)
```

Comment:
@pthierry38 working on `dh_make` at the moment, but I didn't actually know there was a goal to support generation of .deb files until now.